packages/zipkin-transport-aws-sqs/src/AwsSqsLogger.js

```
@@ -0,0 +1,93 @@
+const AWS = require('aws-sdk');
+const {jsonEncoder: {JSON_V2}} = require('zipkin');
+const {EventEmitter} = require('events');
+
+class AwsSqsLogger extends EventEmitter {
+  constructor(options) {
+    super();
+    this.log = options.log || console;
+    this.delaySeconds = options.delaySeconds || 0;
+    this.pollerSeconds = options.pollerSeconds || 100;
+    this.queue = [];
+    this.queueBytes = 0;
+    this.errorListenerSet = false;
+    this.maxPayloadSize = 256 * 1024; // Max Payload size per message is 256KB, limit from AWS SQS
+    this.queueUrl = options.queueUrl;
```

Since this is mandatory, should we throw an error when this is not passed?